hash/gbcolor.xml

```diff
@@ -28397,4 +28397,59 @@ license:CC0-1.0
 			</dataarea>
 		</part>
 	</software>
+
+
+<!-- Aftermarket games -->
+
+	<software name="grimacesb">
+		<description>Grimace's Birthday (rev 2)</description>
+		<year>2023</year>
+		<publisher>Krool Toys</publisher>
+		<notes><![CDATA[
+Officially licensed free game, released as part of McDonald's "Grimace's Birthday" 2023 campaign.
+ROM name was taken from the original file used by the website's emulator.
+]]></notes>
+		<sharedfeat name="compatibility" value="GBC"/>
+		<part name="cart" interface="gameboy_cart">
+			<feature name="slot" value="rom_mbc5" />
+			<dataarea name="rom" size="1048576">
+				<rom name="bX4TvUeiUEfKRw8bkPGaB.gb" size="1048576" crc="9dee42fe" sha1="071fa179df96950c3c92b0abbe32092b1816ec40"/>
+			</dataarea>
```

Comment:
Please use hexadecimal sizes for new additions to this software list.

Reply:
Fixed.